scripts/install-fabric.sh

```diff
@@ -24,9 +24,14 @@ _arg_comp=('' )
 _arg_fabric_version="2.4.7"
 _arg_ca_version="1.5.5"
 
-ARCH=$(echo "$(uname -s|tr '[:upper:]' '[:lower:]'|sed 's/mingw64_nt.*/windows/')-$(uname -m |sed 's/x86_64/amd64/g')" |sed 's/darwin-arm64/darwin-amd64/g')
+OS=$(uname -s|tr '[:upper:]' '[:lower:]'|sed 's/mingw64_nt.*/windows/')
+ARCH=$(uname -m | sed 's/x86_64/amd64/g' | sed 's/aarch64/arm64/g')
+PLATFORM=${OS}-${ARCH}
+
+# Fabric < 1.0 uses uname -m for architecture.
```

Comment:
Fabric < 1.2 uses uname -m for architecture.

Reply:
If 2.5 is an LTS let's leave the bootstrap.sh.